cvxpy/lin_ops/canon_backend.py

```diff
@@ -1596,7 +1226,7 @@ def rmul(self, lin: LinOp, view: StackedSlicesTensorView) -> StackedSlicesTensor
 
             def func(x, p):
                 if p == 1:
-                    return (stacked_lhs @ x).tocsc()
+                    return (stacked_lhs @ x).tocsr()
```

Comment:
commenting on the important changes.
I suspect the speedups have to do with faster row slicing and for p==1 having csc is not beneficial anyways.

Reply:
This is for rmul.